tools/update-eslint.sh

```diff
@@ -20,7 +20,7 @@ npm install --no-bin-links --production --no-package-lock eslint-plugin-markdown
 cd ../..
 
 # Use dmn to remove some unneeded files.
-npx dmn@2.1.0 -f clean
+npx dmn@2.2.1 -f clean
```

Comment:
maybe we want switch to `dmn@latest`. I guess breaking changes would be rather unlikely on it.

Reply:
I'm not the only person who has publishing rights to `dmn, so I'd rather be very conservative with a tool that can run arbitrary commands on a developer's machine. So I'd prefer to pin to a specific version and accept the annoyance of a little churn at each update.

To be clear: I don't have any reason whatsoever to distrust the original maintainer at all. They've been totally A++ 💯 🥇 😍 ✨ awesome. But there are lots of possibilities here: Their account (or my account!) gets hacked. Or they (or I!) in good faith give publishing rights to someone who is a bad actor without knowing it. And so on.

Reply:
True, there is a certain remaining risk, even if you're the maintainer. If you don't mind updating this version, keep it that way. 😉